.chloggen/signalfx-contract.yaml

```diff
@@ -0,0 +1,24 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)component: awsfirehosereceiver
+component: signalfxreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Follow receiver contract based on type of error
+# One or more tracking issues or pull requests related to the change
+issues: [5909]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
```suggestion
subtext: Use 503 error code for retryable and 400 error code for not-retryable errors instead of responding with a 500 unconditionally.
```